app/views/overview/_service-group.html

```diff
@@ -50,14 +50,16 @@ <h4 class="route-title truncate">
           <!--   <span class="pficon pficon pficon-add-circle-o" aria-hidden="true"></span> -->
           <!-- </div> -->
           <h2>No linked services.</h2>
-          <p>
-            No services are linked to {{service.metadata.name}}.
-            <span ng-if="(services | hashSize) > 1">Add a service to group them together.</span>
-          </p>
-          <div ng-if="(services | hashSize) > 1">
-            <button class="btn btn-primary" ng-show="(services | hashSize) > 1" ng-click="linkService()">
-              Link Service
-            </button>
+          <div ng-if="'services' | canI : 'update'">
```

Comment:
@jwforres 
![screenshot](https://cloud.githubusercontent.com/assets/1668218/16726490/c08d3da2-475c-11e6-931d-a6117c734d2d.png)
vs.
![screenshot-2](https://cloud.githubusercontent.com/assets/1668218/16726496/c6f39f38-475c-11e6-90c1-f22673457c44.png)

Thinking about it now I would leave the  "No services are linked to foo." there. Thoughts ?